R/constants.R

```diff
@@ -0,0 +1,20 @@
+CAS_GENOMES <-
+  c("hg38", "hg19", "mm10", "tair10", "custom", "dm6", "sacCer3")
+
+BASIC_GENOMES <-
+  c("hg38",
+    "hg19",
+    "mm10",
+    "tair10",
+    "rhos",
+    "custom",
+    "dm6",
+    "sacCer3")
+
+get_cas_genomes <- function() {
```

Comment:
please move both these functions to R/genomeSpec.R and add documentation.

Reply:
all done